version.go

```diff
@@ -10,4 +10,4 @@ const Version = "0.5.4"
 // A pre-release marker for the version. If this is "" (empty string)
 // then it means that it is a final release. Otherwise, this is a pre-release
 // such as "dev" (in development), "beta", "rc1", etc.
-const VersionPrerelease = ""
+const VersionPrerelease = "de"
```

Comment:
Geeeez. You'd think I'd be extra careful this week... I also forgot to bump the version. Fixed.